src/index.js

```diff
@@ -33,7 +29,7 @@ module.exports = Metalsmith;
  * @param {String} directory
  */
 
-function Metalsmith(directory){
+export function Metalsmith(directory){
```

Comment:
I think you probably want `export default` here. That should resolve the backwards compatibility issue I mentioned.